app/commands.py

```diff
@@ -765,3 +765,31 @@ def create_user_jwt(token):
     service_id = token[-73:-37]
     api_key = token[-36:]
     print(create_jwt_token(api_key, service_id))
+
+
+def update_template(id, name, template_type, content, subject):
```

Comment:
[suggestion] if this is only intended for use from `update_templates`, can we rename it `def _update_template`?